tests/python/relay/test_any.py

```diff
@@ -18,27 +18,101 @@
 
 import tvm
 from tvm import relay
-from tvm.relay import Kind, transform
 from tvm.relay.loops import while_loop
 from tvm.relay.testing import run_infer_type as infer_type
 
 def int32(val):
     return relay.const(val, 'int32')
 
+def verify_any_broadcast(x_shape, y_shape, x_np_shape, y_np_shape, op, np_op):
+    dtype = 'float32'
+    x = relay.var('x', shape=x_shape, dtype=dtype)
+    y = relay.var('y', shape=y_shape, dtype=dtype)
+    mod = relay.module.Module()
+    mod["main"] = relay.Function([x, y], op(x, y))
+    x_np = np.random.uniform(size=x_np_shape).astype(dtype)
+    y_np = np.random.uniform(size=y_np_shape).astype(dtype)
+    for kind in ["debug", "vm"]:
+        ex = relay.create_executor(kind, mod=mod, ctx=tvm.cpu(), target="llvm")
+        result = ex.evaluate()(x_np, y_np)
+        tvm.testing.assert_allclose(result.asnumpy(), np_op(x_np, y_np))
+
+def test_any_broadcast():
+    verify_any_broadcast((relay.Any(),), (3, 2), (1,), (3, 2), relay.add, np.add)
+    verify_any_broadcast((relay.Any(), 2), (1, 2), (1, 2), (1, 2), relay.add, np.add)
+    verify_any_broadcast((relay.Any(), 2), (1, 2), (3, 2), (1, 2), relay.add, np.add)
+    verify_any_broadcast((relay.Any(), 2), (1, 2), (4, 2), (1, 2), relay.add, np.add)
```

Comment:
Is this the same to the last test? It looks that only (3, 2) and (4, 2) are different.

Reply:
removed redundant test